core/src/epicli/data/common/ansible/playbooks/roles/postgres_exporter/tasks/main.yml

```diff
@@ -0,0 +1,111 @@
+---
+- name: Install pip3 [Debian]
```

Comment:
Is it possible to use [package ](https://docs.ansible.com/ansible/latest/modules/package_module.html) module once for this one and the next tasks? Is its name correct?

Reply:
I used this way of installation to be able add --no-install-recommends option - - i don't need them. This should secure me from getting issue on future version if dependencies will change.  I don't see in docs how to use this option with package module.